scaleway/resource_object_bucket.go

```diff
@@ -533,8 +533,6 @@ func resourceScalewayObjectBucketRead(ctx context.Context, d *schema.ResourceDat
 
 	_ = d.Set("cors_rule", flattenBucketCORS(corsResponse))
 
-	_ = d.Set("endpoint", fmt.Sprintf("https://%s.s3.%s.scw.cloud", bucketName, region))
```

Comment:
It seems like a breaking change :/ if people are using this field, this will break their code.

Reply:
Nop, check a few lines above